src/workloads/serverless/IndexStress.yml

```diff
@@ -28,6 +28,24 @@ Clients:
   Client4:
     QueryOptions:
       maxPoolSize: 250
+  Client5:
+    QueryOptions:
+      maxPoolSize: 250
```

Comment:
Was this the maxPoolSize when we ran it successfully yesterday? I remember decreasing it to 50 at some point, but I don't remember at what point..

Reply:
I think we can safely keep the maxPoolSize at 250. As long as it's lower than 500 (which is the max connections per serverless instance), we won't hit any limits.

Reply:
Sounds good!